### PR TITLE
build(client): publish experimental presence

### PR DESCRIPTION
**Pull request by jason-ha**
+ Note current Notifications limitation
+ Connect docs to readme with more details

### CODE REVIEW

**Review by CraigMacomber — APPROVED** [review]

**Comment by CraigMacomber** [comment]
I think you might need a PR description to make to make CI happy.

**Comment by CraigMacomber** [comment]
If you want customers to try this out, then you should probably include a changeset so it will be announced in the release notes.

**Comment by jason-ha** [comment]
> I think you might need a PR description to make to make CI happy.

Yep - that was actually intentional because there are two changes that should happen first. Wanted this one filed for tracking release cut. :)

File: .changeset/moody-rats-cover.md [hunk 1]
```diff
@@ -0,0 +1,16 @@
+---
+"@fluid-experimental/presence": major
+---
+---
+"section": feature
+---
+
+Experimental Presence package added
+
+**[@fluid-experimental/presence](https://github.com/microsoft/FluidFramework/tree/main/packages/framework/presence#readme)** is now available for investigation. The new package is meant to support presence of collaborators connected to the same container. Use this library to quickly share simple, non-persisted data among all clients or send/receive fire and forget notifications.
```
Comment (alexvy86) [inline .changeset/moody-rats-cover.md:10]: I'm not up to date with the exact capabilities of experimental presence, so I read the README, and wanted to confirm if the "_quickly share simple, non-persisted data among all clients_" has guaranteed delivery so all clients (eventually) agree on that state? Based on my little knowledge about experimental presence, I had it very tied to signals and non-guaranteed delivery. If that were the case, I'd suggest making it more explicit in the changeset, but I suspect that's just my outdated understanding.

File: packages/framework/presence/README.md [hunk 2]
```diff
@@ -82,6 +82,11 @@ presence.getStates("app:v1states", { myState2: Latest({x: true})});
  would be compatible with both of the prior schemas as "myState2" is a different name. Though in this situation none of the different clients would be able to observe each other.
 
 
+### Notifications
+
+Notifications API is not well connected. All messages are always broadcast even if `unicast` API is used and all are emitted as `unattendedNotification` event rather than the appropriate custom event.
```
Comment (alexvy86) [inline packages/framework/presence/README.md:87]: I might suggest dropping the first sentence. For a while I couldn't quite make sense of it even in the context of the second one, and I think the second one on its own is clear about the limitation.

**Review by alexvy86 — APPROVED** [review]
Docs look good overall, but left a few comments.

File: .changeset/moody-rats-cover.md (hunk 1 above)
Comment (jason-ha) [reply .changeset/moody-rats-cover.md:10]: Good note to have. I'm adding reliability notes to readme. I think we are not quite as bad as fully unreliable as we think we isolated the most common drop situation and have done some accounting.

File: packages/framework/presence/README.md (hunk 2 above)
Comment (jason-ha) [reply packages/framework/presence/README.md]: 👍 Reworded.

**Pull request merged**